Source/particles/CastroParticles.cpp

```diff
@@ -236,20 +200,17 @@ Castro::TimestampParticles (int ngrow)
     {
 	first = false;
 
-	ParmParse ppp("particles");
 
 	// have to do it here, not in read_particle_params, because Density, ..., are set after
 	// read_particle_params is called.
 
 	int timestamp_density = 1;
```

Comment:
we also need to remove these default value settings -- these will now be done elsewhere in the code